proto/js/wb-checklist.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @title WET-BOEW Hello world plugin
+ * @overview Plugin contained to show an example of how to create your custom WET plugin
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, wb ) {
+  "use strict";
+  /*
+   * Variable and function definitions.
+   * These are global to the plugin - meaning that they will be initialized once per page,
+   * not once per instance of plugin on the page. So, this is a good place to define
+   * variables that are common to all instances of the plugin on a page.
+   */
+  var componentName = "wb-calc",
+      selector = "." + componentName,
+      initEvent = "wb-init" + selector,
+      $document = wb.doc,
+      /**
+       * @method init
+       * @param {jQuery Event} event Event that triggered the function call
+       */
+      init = function( event ) {
+        // Start initialization
+        // returns DOM object = proceed with init
+        // returns undefined = do not proceed with init (e.g., already initialized)
+        var elm = wb.init( event, componentName, selector ),
+            $elms,
+            Checkbox = function (domNode) {

```

Comment:
@delisma not sure this is a good idea. Object.freeze is pretty definitive since it locks the object. May want to revisit